client/pom.xml

```diff
@@ -166,5 +166,15 @@
             <artifactId>oshi-core</artifactId>
             <version>5.3.2</version>
         </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
+            <version>1.5.4</version>
+        </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
+            <version>1.5.4</version>
```

Comment:
In order for Dependabot to be able to automatically update this library, the version number needs to be factored out into a Maven property as in jenkinsci/email-ext-plugin#229. Please also update this to the latest version (1.6.0).

Reply:
See 0c282d774c853da87d748f5ebcc9ba494e5c7138.